libcst/codemod/_cli.py

```diff
@@ -55,7 +55,7 @@ def invoke_formatter(formatter_args: Sequence[str], code: AnyStr) -> AnyStr:
         AnyStr,
         subprocess.check_output(
             formatter_args,
-            env={},
+            env=None,
```

Comment:
just removing this parameter has the same effect as passing in `None`, so let's remove it

Reply:
sure 👌🏻